__factories__/data/firebase.js

```diff
@@ -0,0 +1,18 @@
+import {Factory} from 'rosie';
+
+export const firebaseRepositoryFactory = new Factory().attrs({
```

Comment:
I’d probably give this a name that makes it clear that it is _project_ data stored in Firebase (since there are several other data types/shapes we store in Firebase alongside it)